mmv1/products/vertexai/terraform.yaml

```diff
@@ -34,6 +35,28 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       region: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+  # Vertex AI Endpoint
+  Endpoint: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: false
+    skip_sweeper: true
+    timeouts: !ruby/object:Api::Timeouts
```

Comment:
Let's just make it 10 for all 3. The only time we hit this is if the API's going slowly, or a request gets dropped so the harm in guessing over is low.